src/main/java/io/algopreorg/alog/containsduplicate/Solution3.java

```diff
@@ -0,0 +1,13 @@
+package io.algopreorg.alog.containsduplicate;
+
+public class Solution3 {
+    public boolean containsDuplicate(int[] nums) {
```

Comment:
this case will not work [1,2,3,1] expected true

Reply:
Fixed